### DISCUSSION TITLE
authorize the validator please

### DISCUSSION THREAD

**Discussion by Yurbason** [kcc-community/any-advice-issue#5] [General]
request to authorize the validator
address: 0x0B078059Fd0691CA9eaA13DfdA93F0C6b3C70fA8

**Comment by karpiyksi** [comment]
request to authorize the validator
0x184F6bFD1B9699d3FDf6A287C392DdD9749ce66f

**Comment by viaweb3** [comment]
copy, will reply later

**Comment by Miluer-tcq** [comment]
#授权验证器
地址的请求：0xD42f7148D63e567126f3F436B745C3ADa2F2aFA6

**Comment by viaweb3** [comment]
You need to run a full node before you can be voted as a validator, and please leave your public IP address, so I can check if your node is running or not.

**Reply by karpiyksi** [reply]
IP:178.219.249.122
0x184F6bFD1B9699d3FDf6A287C392DdD9749ce66f

**Reply by Yurbason** [reply]
IP: 65.21.7.227

**Reply by litaochen** [reply]
Please add our validator too.
wallet address: 0xB0a8B8480130E373Dd19a560327Ae0b60C3B2b2D
ip: 34.231.109.252

**Comment by razumv** [comment]
Hello. I am from DoubleTop community. Please add our validator too.
wallet address: 0xe868bE65C50b61E81A3fC5cB5A7916090B05eb2A
ip: 104.248.80.252

**Comment by viaweb3** [comment]
https://gitter.im/kcc-community/governance. please join this room.
